kudo-operator/docs/latest/configuration.md

```diff
@@ -1,2 +1,77 @@
+## Docker
+Docker images used by KUDO Spark and image pull policy can be specified by providing the following parameters:
+```bash
+kubectl kudo install spark --instance=spark-operator \
+        -p operatorImageName=mesosphere/kudo-spark-operator \
+        -p operatorVersion=latest \
+        -p imagePullPolicy=Always
+```
+
+## Monitoring
+Metrics reporting is enabled by default and can be disabled by providing `-p enableMetrics=false` parameter:
+```bash
+kubectl kudo install spark --instance=spark-operator -p enableMetrics=false
+```
+When enabled, KUDO Spark installs Metrics Services and Service Monitors to enable metrics scraping by Prometheus. More
+information about how KUDO Spark exposes metrics and metrics configuration  is available in [KUDO Spark Operator Monitoring](./monitoring.md)
+documentation.
+
+## Spark History Server
+KUDO Spark comes bundled with Spark History Server which is disabled by default. It can be enabled via `-p enableHistoryServer=true`:
+```bash
+kubectl kudo install spark --instance=spark-operator -p enableHistoryServer=true
+```
+
+More information about Spark History Server configuration is available in [Spark History Server Configuration](./history-server.md)
+documentation.
+
+## Service Accounts and RBAC
+By default, KUDO Spark will create service accounts, roles, and role bindings with preconfigured permissions required
+for running Spark Operator and submitting Spark Applications.
+
+### Service Accounts
+The following parameters configure service account settings for Spark Operator:
+```bash
+kubectl kudo install spark --instance=spark-operator \
+        -p operatorServiceAccountName=<service account name> \
+        -p createOperatorServiceAccount=<true|false>
+```
+To use existing service account for the Operator, `createOperatorServiceAccount` parameter should be set to `false`.
+
+Service account configuration for Spark Applications follows the same pattern:
+```bash
+kubectl kudo install spark --instance=spark-operator \
+        -p sparkServiceAccountName=<service account name> \
+        -p createSparkServiceAccount=<true|false>
+```
+To use existing service account for Spark, `createSparkServiceAccount` parameter should be set to `false`.
+
+### RBAC
+KUDO Spark `ClusterRole` with necessary permissions created automatically, but this can be disabled by providing `-p createRBAC=false` parameter:
+```bash
+kubectl kudo install spark --instance=spark-operator -p createRBAC=<true|false>
+```
+
+KUDO Spark requires a `ClusterRole` to be configured in order for it to handle custom resources, listen to events, work
+with secrets, configmaps, and services. The full list of permissions is available in [spark-operator-rbac.yaml](../../operator/templates/spark-operator-rbac.yaml).
+If `createRBAC` parameter is set to `false`, a `ClusterRole` should be created or exist before the installation alongside with a
```

Comment:
```suggestion
If `createRBAC` parameter is set to `false`, a `ClusterRole` should be created or exist before the installation alongside a
```

Reply:
thanks for catching this, the wording in the whole sentence doesn't look right. rephrased it.